test_merge_sort.py

```diff
@@ -0,0 +1,28 @@
+from random import shuffle
+import pytest
+
+from merge_sort import merge_srt
+
+
+def test_merge_sort():
+    expected = range(20)
+    unsorted = expected[:]
+    shuffle(unsorted)
+    merge_srt(unsorted)
+    actual = unsorted
```

Comment:
Makes sense. I think the rebind here was just for readability sake, but, upon reflection, it is unnecessary.